src/braindrop/app/widgets/raindrop_details.py

```diff
@@ -265,7 +265,7 @@ def _refresh_display(self) -> None:
             self._set(
                 "collection",
                 f"{PUBLIC_ICON if self.data.collection(self.raindrop.collection).public else PRIVATE_ICON}"
-                f" {self.data.collection(self.raindrop.collection).title}",
+                f" {escape(self.data.collection(self.raindrop.collection).title)}",
```

Comment:
As noted in a comment in #171 this wouldn't seem to actually address what you think is the cause of the problem. Let's keep this here anyway -- it's sensible that this does get escaped and it always should have been -- but I think we might want to add a couple more calls to `escape` elsewhere too, depending on what problem you're actually seeing and what the actual cause is.